demo/image_demo.py

```diff
@@ -25,7 +25,9 @@ def parse_args():
     return args
 
 
-def main(args):
+def main():
```

Comment:
Is this demo necessary after the bottom-up models have been supported?

Reply:
I think we can keep it for now in case someone wants to inference the model on cropped images.